src/main/resources/reference.conf

```diff
@@ -46,4 +46,26 @@ videobridge {
     # Must be set when enabled = true
     # domain="some-domain"
   }
+  ice {
+    tcp {
+      # Whether ICE/TCP is enabled.
+      enabled = false
+      port = 443
+      # Port to fallback to in case binding to 'port' fails. Set to -1 to disable fallback.
+      fallback-port = 4443
+      # An optional additional port to advertise.
+      # mapped-port = 8443
+      # Whether to use "ssltcp" or plain "tcp".
+      ssltcp = true
+    }
+    # The port for ICE/UDP.
+    port = 10000
```

Comment:
Should we name the config value something more descriptive to be more self-documenting?  Or nest it within a `UDP` block?